mage_ai/orchestration/db/models/oauth.py

```diff
@@ -31,6 +35,7 @@ class User(BaseModel):
 
     oauth2_applications = relationship('Oauth2Application', back_populates='user')
     oauth2_access_tokens = relationship('Oauth2AccessToken', back_populates='user')
+    roles_new = relationship('Role', secondary='user_role', back_populates='users')
```

Comment:
Could you add comment for the roles and roles_new? Would user_roles be a better name here?